.github/workflows/integration_tests.yml

```diff
@@ -55,6 +71,7 @@ jobs:
         with:
           KEPTN_VERSION: ${{ matrix.keptn-version }}
           KUBECONFIG: /etc/rancher/k3s/k3s.yaml
```

Comment:
I think this variable is redundant since it is set above for the whole workflow :)

Reply:
Nice catch! The other failures still need to be debugged and will be fixed in a separate PR.